dbt_invoke/properties.py

```diff
@@ -206,19 +208,29 @@ def echo_macro(ctx):
         f'\n{_utils.get_macro(_MACRO_NAME)}'
     )
 
+@functools.lru_cache
```

Comment:
My knowledge on `@functools.lru_cache` is limited, but is it helping us here given that we only run `_read_manifest` once in each invocation of the `migrate` command?

Reply:
I think my prior cut was calling `_read_manifest` more than once so I likely just neglected to remove it. I will add a change for this when I push the commit for deletion of existing yml files.